app/parse.py

```diff
@@ -7,9 +17,61 @@
     author: str
     tags: list[str]
 
+QUOTES_FIELDS = [field.name for field in fields(Quote)]
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)8s]: %(message)s",
+    handlers=[
+        logging.FileHandler(os.path.join("parser.log")),
+        logging.StreamHandler(sys.stdout),
+    ],
+)
+
+def parse_single_quote(quote) -> Quote:
+    tags_soup = quote.select_one(".tags")
+    tags = [tag.text for tag in tags_soup.select(".tag")]
+    return Quote(
+        text=quote.select_one(".text").text,
+        author=quote.select_one(".author").text,
+        tags=tags
+    )
+
+def get_single_page_quotes(page_soup: BeautifulSoup) -> [Quote]:
+    quotes = page_soup.select(".quote")
+    return [parse_single_quote(quote_soup) for quote_soup in quotes]
+
+def get_list_of_quotes() -> list[Quote]:
+    logging.info("Start parse")
+    quote_page = requests.get(BASE_URL).content
+    first_page_soup = BeautifulSoup(quote_page, "html.parser")
+
+    all_quotes = get_single_page_quotes(first_page_soup)
+    logging.info("First page was parsed successfully")
+
+    next_btn = first_page_soup.select_one(".next")
+    print(next_btn)
```

Comment:
The `print` statement here seems to be used for debugging purposes. If it's not needed for the final implementation, consider removing it to keep the code clean.